src/legacy/core_plugins/kibana/public/management/index.scss

```
@@ -11,5 +11,5 @@
 
 // Core
 @import 'management_app';
-@import '../../../../../plugins/advanced_settings/public/management_app/advanced_settings';
+@import '../../../../../plugins/advanced_settings/public';
```

SASS doesn't automatically find `index` files. This import will fail. It needs to be 
```suggestion
@import '../../../../../plugins/advanced_settings/public/index';
```